test/browser.js

```diff
@@ -171,27 +172,48 @@ test('throws if does not support ReadableStream', withPage, async (t, page) => {
 	await server.close();
 });
 
-test('FormData with searchParams', withPage, async (t, page) => {
-	t.plan(2);
-
+test.failing('FormData with searchParams', withPage, async (t, page) => {
+	t.plan(6);
 	const server = await createTestServer();
 	server.get('/', (request, response) => {
 		response.end();
 	});
 	server.post('/', async (request, response) => {
-		const requestBody = await pBody(request);
-
-		t.regex(requestBody, /bubblegum pie/);
 		t.deepEqual(request.query, {foo: '1'});
+		const body = await new Promise((resolve, reject) => {
+			const busboy = new Busboy({headers: request.headers});
+
+			busboy.on('file', (fieldname, file, filename, encoding, mimetype) => {
+				let fileContent = '';
+
+				file.on('data', data => {
```

Comment:
You should be able to use async iteration on the `file` stream.

```js
for await (const chunk of file) {
    fileContent += chunk;
}
```

This may simplify things, as errors will be thrown automatically and you should be able to `resolve()` after the loop without having to put it in an `end` callback.

Read more here:
https://2ality.com/2019/11/nodejs-streams-async-iteration.html#helper-function%3A-readabletostring()